design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -96,13 +97,15 @@ input DeleteNegotiableQuotesInput {
 }
 
 type DeleteNegotiableQuotesOutput {
+    status: Boolean!
```

Comment:
According to discussion with the PWA team, this field should be changed to an enum with (at least) three states: success, failure, and partial success.  Also rename to something more descriptive, possibly `resultState`?

Reply:
Related: https://jira.corp.magento.com/browse/PWA-1523 will add expanded error information to the output and allow callers to easily grab the list of failing ids when there's a partial success.